src/irmin-pack/irmin_pack.ml

```diff
@@ -399,6 +399,52 @@ struct
     end
   end
 
+  let integrity_check (t : X.Repo.t) =
+    Fmt.pr "running the integrity check\n%!";
```

Comment:
This should be using `Logs` instead of unconditionally printing things on stdout (maybe `Log.info` in that case)